.github/workflows/release.yml

```
@@ -2,26 +2,169 @@ name: Release
+        run: npm ci
+
+      - name: Build
+        run: npm run build:lib
+
+      - name: Publish release candidate
+        run: |
+          npm exec --workspaces -- ../../packages/repo-configs/scripts/prepare-release-candidate
+          npx changeset publish --tag next
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Output canary version
+        uses: actions/github-script@v4.0.2
+        with:
+          script: |
+            const paths = ['react', 'design-tokens', 'css'].forEach((path) => {
+              const package = require(`${process.env.GITHUB_WORKSPACE}/packages/${path}/package.json`)
+              github.repos.createCommitStatus({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                sha: context.sha,
+                state: 'success',
+                context: `Published ${package.name}`,
+                description: package.version,
+                target_url: `https://unpkg.com/${package.name}@${package.version}/`
+              })
+            })
+
+  release-candidate-next-minor:
+    if: ${{ github.repository == 'primer/brand' && github.ref_name == 'changeset-release/next-minor' }}
+    name: Next minor candidate (@next-minor)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Set up Node
+        uses: actions/setup-node@v6
+        with:
+          node-version: 24
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build:lib
+
+      - name: Publish release candidate
+        run: |
+          npm exec --workspaces -- ../../packages/repo-configs/scripts/prepare-release-candidate
+          npx changeset publish --tag next-minor
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Output canary version
+        uses: actions/github-script@v4.0.2
+        with:
+          script: |
+            const paths = ['react', 'design-tokens', 'css'].forEach((path) => {
+              const package = require(`${process.env.GITHUB_WORKSPACE}/packages/${path}/package.json`)
+              github.repos.createCommitStatus({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                sha: context.sha,
+                state: 'success',
+                context: `Published ${package.name}`,
+                description: package.version,
+                target_url: `https://unpkg.com/${package.name}@${package.version}/`
+              })
+            })
+
+  release-canary:
+    if: ${{ github.repository == 'primer/brand' && github.ref_name != 'main' && github.ref_name != 'next-minor' && github.ref_name != 'changeset-release/main' && github.ref_name != 'changeset-release/next-minor' }}
```

This one feels pretty verbose now. How about we flatten the changeset branch conditions with [startsWith](https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#startswith)

```suggestion
    if: ${{ github.repository == 'primer/brand' && github.ref_name != 'main' && github.ref_name != 'next-minor' && !startsWith(github.ref_name, 'changeset-release/') }}

```